<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SalesWindow</class>
 <widget class="QMainWindow" name="SalesWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Продажи</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="productsGroup">
      <property name="title">
       <string>Товары</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="searchInput">
           <property name="placeholderText">
            <string>Поиск...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="searchButton">
           <property name="text">
            <string>Найти</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="productsTable">
         <column>
          <property name="text">
           <string>Название</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Цена</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>В наличии</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Кол-во:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="quantitySpin">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="addButton">
           <property name="text">
            <string>Добавить</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="cartGroup">
      <property name="title">
       <string>Корзина</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTableWidget" name="cartTable">
         <column>
          <property name="text">
           <string>Товар</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Цена</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Кол-во</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Сумма</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="removeButton">
           <property name="text">
            <string>Удалить</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="clearButton">
           <property name="text">
            <string>Очистить</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="totalLabel">
         <property name="text">
          <string>Итого: 0 руб</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="customerInput">
         <property name="placeholderText">
          <string>Имя покупателя</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="checkoutButton">
         <property name="text">
          <string>Оформить продажу</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>